src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -215,6 +215,26 @@ private MenuItem SearchForMenuItem(MenuBarType type)
             }
         }
 
+        /// <summary>
+        /// Request to open a view extension in the side panel by name.
+        /// </summary>
+        /// <param name="extensionName"></param>
+        public void OpenViewExtension(string extensionName)
+        {
+            dynamoViewModel.OnViewExtensionOpenRequest(extensionName);
```

Comment:
are extension names localized? If so this could get tricky - if not seems fine.

Reply:
This name is not localized typically but could be.  We can change it to the UniqueID.